tests/aws/services/stepfunctions/v2/error_handling/test_task_lambda.py

```diff
@@ -53,6 +48,40 @@ def test_raise_exception(
             exec_input,
         )
 
+    @markers.aws.validated
+    def test_raise_custom_exception(
```

Comment:
Should we add a JSONata test to this as well in a follow-up? Curious if the `$states.errorOutput` works here too.